fairlearn/reductions/_grid_search/_grid_generator.py

```diff
@@ -20,11 +20,36 @@
 class _GridGenerator:
     """A generator of a grid of points with a bounded L1 norm."""
 
-    def __init__(self, grid_size, grid_limit, pos_basis, neg_basis, neg_allowed, force_L1_norm):
+    def __init__(self, grid_size, grid_limit, pos_basis, neg_basis, neg_allowed, force_L1_norm,
+                 grid_offset=None):
+        """Initialize with grid generator utility.
+
+        The `grid_size` is the number of columns to be generated in the grid.
+        :type grid_size: int
+
+        The `grid_limit` is the range of the values in the grid generated.
+        :type grid_limit: float
+
+        The `neg_allowed` ensures if we want to include negative values in the grid or not.
+        If True, the range is doubled.
+        :type neg_allowed: boolean
+
+        The `force_L1_norm`, if True, ensures that all points of the grid have the L1 norm equal
+        to grid_limit. If False, then grid consists of points whose L1 norm is less than equal to
+        grid_limit.
+        :type force_L1_norm: boolean
+
+        The `grid_offset` shifts the whole grid by that value.
+        :type grid_offset: float
```

Comment:
Isn't it rather a vector than a scalar?

Reply:
Yes. My bad. I'll correct the documentation.